src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -231,7 +232,10 @@ class AddUserProfile extends Component {
                         <option value="Volunteer">Volunteer</option>
                         <option value="Manager">Manager</option>
                         <option value="Core Team">Core Team</option>
-                        {/* </select> */}
+                        <option value="Mentor">Mentor</option>
+                        {hasPermission(this.props.auth.user.role, 'addDeleteEditOwners') && (
```

Comment:
Could you explain here why we need a `owner` check here? I'm assuming if the requestor's role is not `owner`, he wouldn't see the `owner` option in the `Role` dropdown. But I logged as an `admin` but still can see the `owner` option there 
<img width="496" alt="image" src="https://user-images.githubusercontent.com/5071040/168397343-c50cbffd-0d1c-48a2-a3f5-147807c8de25.png">

Reply:
Nice catch! Will update!